.spelling

```diff
@@ -193,6 +196,7 @@ parameterizing
 params
 pprof
 pre-commit
+prometheus
```

Comment:
it seems like this isn't needed if this is properly capitalized everywhere? It should be capitalized, as it is a proper noun

Reply:
Done.